scripts/validate_docstrings.py

```diff
@@ -497,6 +497,44 @@ def parameter_desc(self, param):
                 desc = desc[: desc.index(full_directive)]
         return desc
 
+    def parameter_desc_list(self, param):
+        """Return a parameter description as a list.
+
+        Each line of the parameter docstring is a string in the list.
+        """
+        # The 'Parameters' value of NumpyDocString._parsed_data is a list of
+        # NamedTuples.
+        parsed_param = [
+            p for p in self.doc._parsed_data["Parameters"] if p.name == param
+        ][0]
+        desc_list = [line for line in parsed_param.desc if line]
+        # Find and strip out any sphinx directives
+        directives_pattern = re.compile(".. ({})".format("|".join(DIRECTIVES)))
```

Comment:
Can you just use the `DIRECTIVE_PATTERN` global that already exists?

Reply:
Hi @WillAyd , `DIRECTIVE_PATTERN` doesn't seem to identify correct directives. It only matches directives if they are _not_ followed by 2 colons (e.g. `.. versionadded 0.1.2`)

Reply:
Can you try and fix the global then? If there's a problem with that should fix directly rather than creating something new.

Also note I *think* this one is incorrect as the periods aren't escaped, so it would literally match any two items to start

Reply:
Yes, you are right. I think `DIRECTIVE_PATTERN` should probably be fixed in another PR since it has some tests related to it.